src/lib/Employee.js

```diff
@@ -0,0 +1,26 @@
+class Employee {
+  constructor(name, id, email) {
```

Comment:
To set this up to receive an object as explained in the comment in the index.js file, we would see

```
class Employee {
    constructor( {name, id, email} ) {
 // rest of code as is
```